src/main/java/frc/robot/subsystems/Turret.java

```diff
@@ -22,6 +24,9 @@
   private WPI_TalonSRX mTurretTalon;
   private Potentiometer mPot;
 
+  // TODO: this
```

Comment:
What is "this" TODO?

Reply:
@onedeadeye > This was an accidental leftover from when I didn't realize we had a constant already defined for the camera ring PWM ID.